src/actor/actor-document.js

```diff
@@ -61,9 +74,23 @@ export class FabulaUltimaActor extends Actor {
 // 	get unlimitedPush() {
 // 		return this.getFlag("fabula-ultima", "unlimitedPush") ?? false;
 // 	}
-//
-// 	/* Override */
-// 	getRollData() {
+  //
+  /* Override */
+  getRollData() {
+    return {
+      dexterity: this.dexterity,
+      insight: this.insight,
+      might: this.might,
+      willpower: this.willpower,
+      dex: `1d${this.dexterity}`,
+      ins: `1d${this.insight}`,
+      mig: `1d${this.might}`,
+      wlp: `1d${this.willpower}`,
+      hp: this.hitPoints,
+      mp: this.mindPoints,
+      ip: this.inventoryPoints,
+    }
+  }
 // 		return {
```

Comment:
NB: It looks like we still have the old version of this function body. Can we remove this?